packages/web/src/utils/route.ts

```diff
@@ -345,3 +348,21 @@ export const pushUniqueRoute = (route: string) => {
   }
   return { type: '' }
 }
+
+export const getEntityLink = (
+  entity: (Track & { user: User }) | (Collection & { user: User }),
```

Comment:
Feels like we should break `NotificationEntity` into its own type

Reply:
yeah should be reusing EntityType, good callout